{"name": "Проверка перехода к разделу  \"История заказов\" под пользователями: без заказов и с заказами", "status": "passed", "description": "Открываем сайт, переходим на страницу входа и логинимся. нажимаем кнопку \"Личный кабинет\" и переходим на вкладку \"История заказов\". Проверяем наличие контейнера заказов", "steps": [{"name": "Переход в личный кабинет с авторизацией", "status": "passed", "steps": [{"name": "Клик по элементу с обработкой исключения", "status": "passed", "steps": [{"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 25, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481147, "stop": 1726773481179}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481147, "stop": 1726773481194}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 25, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481219, "stop": 1726773481285}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481219, "stop": 1726773481294}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 25, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481325, "stop": 1726773481353}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481323, "stop": 1726773481364}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1176,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 25, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481396, "stop": 1726773481435}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481396, "stop": 1726773481444}, {"name": "поиск элемента с нажатием на него", "status": "passed", "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481467, "stop": 1726773481497}], "parameters": [{"name": "locator", "value": "('xpath', './/a[@href=\"/account\"]')"}], "start": 1726773481467, "stop": 1726773481741}], "parameters": [{"name": "button", "value": "('xpath', './/a[@href=\"/account\"]')"}, {"name": "locator_exception", "value": "('xpath', './/*[@class=\"Modal_modal_overlay__x2ZCr\"]')"}], "start": 1726773481147, "stop": 1726773481742}], "start": 1726773481146, "stop": 1726773481742}, {"name": "Переход в раздел \"История заказов\"", "status": "passed", "steps": [{"name": "Клик по элементу с обработкой исключения", "status": "passed", "steps": [{"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"Account_link__2ETsJ text text_type_main-medium text_color_inactive\" href=\"/account/order-history\"> is not clickable at point (128,307) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 25, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726773481743, "stop": 1726773481781}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726773481743, "stop": 1726773481790}, {"name": "поиск элемента с нажатием на него", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"Account_link__2ETsJ text text_type_main-medium text_color_inactive\" href=\"/account/order-history\"> is not clickable at point (128,307) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 25, in click_to_element\n    self.find_element_with_wait(locator).click()\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726773481813, "stop": 1726773481853}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726773481812, "stop": 1726773481870}, {"name": "поиск элемента с нажатием на него", "status": "passed", "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726773481894, "stop": 1726773481928}], "parameters": [{"name": "locator", "value": "('xpath', './/a[text()=\"История заказов\"]')"}], "start": 1726773481894, "stop": 1726773482161}], "parameters": [{"name": "button", "value": "('xpath', './/a[text()=\"История заказов\"]')"}, {"name": "locator_exception", "value": "('xpath', './/*[@class=\"Modal_modal_overlay__x2ZCr\"]')"}], "start": 1726773481742, "stop": 1726773482162}], "start": 1726773481742, "stop": 1726773482162}, {"name": "Поиск контейнера истории", "status": "passed", "steps": [{"name": "Поиск элемента с ожиданием", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', './/div[contains(@class, \"Account_contentBox\")]')"}], "start": 1726773482162, "stop": 1726773482191}], "start": 1726773482162, "stop": 1726773482192}], "attachments": [{"name": "log", "source": "02346f31-fc61-4517-9c47-4a9a06d082f5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "driver", "value": "'firefox'"}, {"name": "customer", "value": "'response_customer'"}], "start": 1726773479877, "stop": 1726773482192, "uuid": "90cd19f1-da16-4f69-a054-b26d85974538", "historyId": "25de67c2bd024c179a78e2e62ebd6486", "testCaseId": "106f9cff248f86c186ec3db8a890e3eb", "fullName": "tests.test_account_page.TestAccountPage#test_transition_order_history", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_account_page"}, {"name": "subSuite", "value": "TestAccountPage"}, {"name": "host", "value": "DESKTOP-Q7I2UDM"}, {"name": "thread", "value": "37340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_account_page"}]}